website/docs/table-group/data-table/data-table.md

```diff
@@ -18,31 +18,26 @@ Our design system has two table types – [primary](../table-primary/table-prima
-| ------------------------- | ---------------------------------------------------------------------------------------------------------------------- |
-| Text in the header        | Use `--fs-100` and `--lh-100` tokens for font-size and line-height respectively, and `--text-primary` token for color. |
-| The text in the row       | Use `--fs-200` and `--lh-200` tokens for font-size and line-height respectively, and `--text-primary` token for color. |
-| Secondary text            | Use `--text-secondary` token for color.                                                                                |
-| Stand alone icon          | Icon with M size and `--icon-secondary-neutral` token for color.                                                       |
-| The icon next to the text | Icon with M size and `--icon-secondary-neutral` token for color.                                                       |
-| Header and cell checkbox  | Use [Checkbox](/components/checkbox/checkbox) with M size.                                                             |
+| Content                                     | Styles and tokens
+| ------------------------------------------- | ---------------------------------------------------------------------
+| The text in the row                         | Use `--fs-200` and `--lh-200` tokens for font-size and line-height respectively, and `--text-primary` token for color.
+| Secondary text                              | Use `--text-secondary` token for color.
+| Icon, either next to the text or standalone | Use M size and `--icon-secondary-neutral` token for color.
 
 ## Sorting
 
-For detailed information on sorting columns in the table, refer to [Table controls](/table-group/table-controls/table-controls).
+For detailed information on sorting columns in the table, refer to [Table controls](/table-group/table-controls/table-controls#sorting).
 
 ## Tooltip
 
-To display additional information about a column, use tooltip. It will appear by hovering over the column title.
+You can use tooltips to display additional information about columns. It will appear by hovering over the column title.
 
-### Conditions for tooltip appearance
-
-Table: Conditions for the tooltip appearance
+Table: Tooltips in Table header
 
 | Case                                                  | Appearance example        |
 | ----------------------------------------------------- | ------------------------- |
-| If text is too long and collapsed into an `ellipsis`. | ![](static/tooltip-1.png) |
-| If the column has additional explanatory information. | ![](static/tooltip-2.png) |
+| Use [Ellipsis](../../components/ellipsis/ellipsis.md) if the column title is too long and needs to be truncated.                        | ![](static/tooltip-1.png) |
+| Use [Tooltip](../../components/tooltip/tooltip.md) if the column has additional explanatory information. | ![](static/tooltip-2.png) |
```

Comment:
Just a quick thought: it might not be super obvious right away what the difference between these two cases is. So I was thinking, maybe we could tweak the illustration in the second row to show the text without an ellipsis? But it's just a thought

<img width="719" alt="image" src="https://github.com/user-attachments/assets/2f74c5a4-a71b-43d4-b5cc-44fca3915ae7" />